testing/test_test.py

```diff
@@ -0,0 +1,63 @@
+from asyncio import format_helpers
+import random
+import pytest
+import numpy as np
+
+from batlib.model import Model
+from batlib.model import PolyStruct
+
+
+def voltage_noise_array(final_value, noise_range, noise_voltage, length):
+    volt_data = [0 for i in range(0, length-1)]
+    volt_data = [(random.uniform(-noise_range, noise_range)+noise_voltage)
+                 for i in range(0, int(length/2)-1)]
+    for i in range(int(length/2), length-1):
+        volt_data.append(final_value)
+    return volt_data
+
+
+def test_test0():
```

Comment:
it's helpful when the test names are more descriptive. Keep in mind that unit tests serve two functions: 1) test code 2) form of documentation (someone can look at the unit tests to see how to use the code)